internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,58 @@
 package daysteps
 
 import (
-	...
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint5-final/internal/personaldata"
+	"github.com/Yandex-Practicum/go1fl-sprint5-final/internal/spentenergy"
 )
 
 const (
 	StepLength = 0.65
 )
 
 // создайте структуру DaySteps
-...
-
+type DaySteps struct {
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
+}
 
 // создайте метод Parse()
-...
+func (ds *DaySteps) Parse(datastring string) (err error) {
+	parts := strings.Split(datastring, ",")
+	if len(parts) != 2 {
+		err = errors.New("error while splitting datastring")
+		return
+	}
+	ds.Steps, err = strconv.Atoi(parts[0])
+	if err != nil {
+		err = errors.New("error while converting string to int")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		err = fmt.Errorf("conversion error: %w", err)
```